pkg/deployments/tasks/00000000-tokens/output/gnosis.json

```diff
@@ -0,0 +1,4 @@
+{
+  "BAL": "0x7eF541E2a22058048904fE5744f9c7E4C57AF717",
+  "WETH": "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d"
```

Comment:
Let's add a readme to  this task explaining that Gnosis uses xDAI as it's native asset so "WETH" is really wrapped DAI. Similarly for the other chains which have their own native asset.

Reply:
These are jsons :( No docs there

Reply:
I added a note in the readme.